x-pack/plugins/search_playground/public/components/message_list/assistant_message.tsx

```diff
@@ -96,6 +106,14 @@ export const AssistantMessage: React.FC<AssistantMessageProps> = ({ message }) =
           username={username}
           timelineAvatar="dot"
           data-test-subj="retrieval-docs-comment-no-docs"
+          eventColor="subdued"
+          css={{
+            '.euiAvatar': { backgroundColor: euiTheme.colors.ghost },
```

Comment:
theres not a better way here for this?

Reply:
Unfortunately not, euiComment has poor flexibility by props, so need to apply these changes
[doc](https://eui.elastic.co/#/display/comment-list)